locales-src/asrouter.ftl

```diff
@@ -90,17 +90,42 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-fx70-hero-title = { -brand-short-name } now fights harder for your privacy
+cfr-whatsnew-fx70-hero-body = The latest update enhances the Tracking Protection feature and makes it easier than ever to create secure passwords for every site.
+
+cfr-whatsnew-fx70-notes-body = Protection reports, Secure Proxy for public Wifi, improved account controls, and many more safety and security tools are included in the latest update.
+cfr-whatsnew-fx70-notes-link-text = Read the release notes
+
+cfr-whatsnew-tracking-protect-title = Protect yourself from trackers
+cfr-whatsnew-tracking-protect-body = { -brand-short-name } blocks many common social and cross-site trackers that follow what you do online.
+cfr-whatsnew-tracking-protect-link-text = View your protection report
+
 # This string is displayed before a large numeral that indicates the total number
 # of tracking elements blocked. Don’t add $blockedCount to
 # your localization, because it would result in the number showing twice.
-cfr-whatsnew-tracking-protections-title =
+cfr-whatsnew-tracking-blocked-title =
```

Comment:
Renamed to `cfr-whatsnew-tracking-blocked-*` to differentiate slightly from the generic numberless version that I named `cfr-whatsnew-tracking-protect-*`